packages/runtime/container-runtime/src/test/gc/gcSummaryStateTracker.spec.ts

```diff
@@ -22,164 +22,13 @@ type GCSummaryStateTrackerWithPrivates = Omit<GCSummaryStateTracker, "latestSumm
 };
 
 describe("GCSummaryStateTracker tests", () => {
-	describe("Summary state reset", () => {
```

Comment:
Love this red. Really nice that the summarizer node `hasChanged` logic is sufficient for all these cases.